vault/resource_database_secret_backend_connection_test.go

```diff
@@ -24,7 +24,7 @@ func TestAccDatabaseSecretBackendConnection_import(t *testing.T) {
 		CheckDestroy: testAccDatabaseSecretBackendConnectionCheckDestroy,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccDatabaseSecretBackendConnectionConfig_postgresql(name, backend, connURL),
+				Config: testAccDatabaseSecretBackendConnectionConfig_postgresql(name, backend, connURL, 0),
```

Comment:
I ran this one locally and got a test failure. Initially it was just about the connection url in plain text not matching a redacted one, so I commented that out (on line 35) and re-ran it and got this:
```
=== RUN   TestAccDatabaseSecretBackendConnection_import
--- FAIL: TestAccDatabaseSecretBackendConnection_import (0.03s)
    testing.go:434: Step 0 error: After applying this step, the plan was not empty:
        
        DIFF:
        
        UPDATE: vault_database_secret_backend_connection.test
          postgresql.0.connection_url: "*****://*****:*****@localhost:5432/*****" => "postgres://postgres:postgres@localhost:5432/postgres"
        
        STATE:
        
        vault_database_secret_backend_connection.test:
          ID = tf-test-db-3758954475378231597/config/db-7907426470007612753
          allowed_roles.# = 2
          allowed_roles.0 = dev
          allowed_roles.1 = prod
          backend = tf-test-db-3758954475378231597
          name = db-7907426470007612753
          postgresql.# = 1
          postgresql.0.connection_url = *****://*****:*****@localhost:5432/*****
          postgresql.0.max_connection_lifetime = 0
          postgresql.0.max_idle_connections = 0
          postgresql.0.max_open_connections = 2
          verify_connection = true
        
          Dependencies:
            vault_mount.db
        vault_mount.db:
          ID = tf-test-db-3758954475378231597
          path = tf-test-db-3758954475378231597
          type = database
```

Reply:
I believe https://github.com/terraform-providers/terraform-provider-vault/pull/217 should resolve this test issue as its also due to the redacted URL.